docs/content/docs/test_framework/Mocking.md

```diff
@@ -52,6 +52,27 @@ class MyAwesomeTests {}
 
 This annotation is available in all types of tests but make sense mostly for functional tests.
 
+## Mocking Preferences
+
+All kind of tests are using real implementations of `com.avito.android.util.preferences.Preferences`, not mocked ones.
+But it's unnecessary to mock `Preferences` somehow because all test data is [cleared]({{< ref "/docs/test/TestDataClearing.md" >}}) between tests,
```

Comment:
`somehow` is redundant

Reply:
Ok. I've tried to be less official. I'll remove this word.